bower.json

```diff
@@ -1,22 +1,21 @@
 {
   "name": "ember-frost-sidebar",
   "dependencies": {
-    "ember": "~2.3.0",
+    "ember": "~2.7.0",
```

Comment:
I tried to increase this to 2.8, but I'm getting a lot of issues with the liquid-fire error. I think I will raise a bug for now to increase Ember to 2.8 and fix this liquid fire error. Will that be ok?